apache/superset
# fix(table-chart): bump legacy-table-chart to 0.11.18

### CATEGORY


### SUMMARY

The new table chart implementation added in #9269 and #9234 throws a JS error when query results are empty but the metric list is not empty. 

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

Before: there is a JS error when query results are empty for a table visualization with metrics:

```text
An error occurred while rendering the visualization: TypeError: Cannot read property '<metric_name>' of undefined
```

After: it should show "no data available in table".
![image](https://user-images.githubusercontent.com/335541/76374629-06e4da80-6301-11ea-8a6e-054249d4d16a.png)


### TEST PLAN

1. Create a new Chart
2. Add some metrics
3. Add a query filter that produces empty results
4. Run the query and check the chart

### ADDITIONAL INFORMATION

N/A

### REVIEWERS

cc: @etr2460 @kristw @graceguo-supercat
